rpgJavaInterpreter-core/src/test/resources/smeup/MUDRNRAPU00249.rpgle

```diff
@@ -0,0 +1,11 @@
+      * Helper declarations
+     D £DBG_Str        S             2
+
+      * Declaring an array field (£6I) in a data structure with an empty INZ keyword.
+      * Should not be assigned to a numeric
+     D £C6MIM          DS
+     D  £6I                          20P 6 DIM(99) INZ                          Importi
```

Comment:
What should be the value of the elements in this array when initialized like this? You should add a stronger assertion, for instance, by testing the value of the first element of the array.